StandaloneViewer/StandaloneViewer/client/components/toolbarSection/toolbarSection.js

```diff
@@ -126,7 +126,7 @@ Template.toolbarSection.helpers({
         const buttonData = [];
 
         buttonData.push({
-            id: 'stackScroll',
+            id: 'stackScrollMouseWheel',
```

Comment:
This button enables left-click dragging, not mousewheel, so this change makes no sense.